cmd/local-gadget/trace/bind.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/cilium/ebpf"
+	"github.com/spf13/cobra"
+
+	commontrace "github.com/kinvolk/inspektor-gadget/cmd/common/trace"
+	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
+	"github.com/kinvolk/inspektor-gadget/cmd/local-gadget/utils"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadget-collection/gadgets/trace"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/bind/tracer"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/bind/types"
```

Comment:
Thanks @invidian for these comments, I think it is very enriching to receive this kind of feedback as sometimes, after working for a while on a project, one gets used to the code, and a fresh and seasoned view is helpful!

> Is there a strong argument for having a `tracer` package or to even differentiate between `tracer` and `types` package?
>
I remember I had this discussion with @alban. IIRC, we said it was necessary because the CLI needs the `types` but we didn't want to import all the Trace CR stuff. However, it was before https://github.com/kinvolk/inspektor-gadget/issues/723, where we divided it into [gadget-collection](https://github.com/kinvolk/inspektor-gadget/tree/main/pkg/gadget-collection) (code related to the Trace CR) and [gadgets](https://github.com/kinvolk/inspektor-gadget/tree/main/pkg/gadgets) (code related to the gadget tracer). Maybe, after that change, it makes sense to put `tracer` and `types` together? It would be indeed more intuitive from the user's point of view. @mauriciovasquezbernal, any comment on this? 

> Alternatively, better variable and import names could also help here, with imports like `bindtracer` or something. Otherwise code starts to look the same everywhere.
>
I could start doing that in `local-gadget`, but I would prefer to create a new issue where we update it everywhere in the project. Do other folks have comments on this?

Reply:
> Is there a strong argument for having a tracer package or to even differentiate between tracer and types package?

The goal is to provide a package that can be imported by applications without having to take all the dependencies on the tracer. For instance, a client that only unmarshals events sent by a server only needs to know the types. Do you think there is another way to implement this?

Reply:
Thanks for considering it @blanquicet. I forgot to mention that this is feedback not strictly related to this PR of course.